frb/figures/finder.py

```diff
@@ -19,6 +19,8 @@
 from astropy.coordinates import ICRS
 from astropy.time import Time
 
+from photutils import SkyRectangularAperture
```

Comment:
This adds a dependency to FRB.  I'm ok with that, but
I think we should make it optional.  And we should add
a note about it in the Install.  Here, wrap it in a `try/except`, 

```
try:
     from photutils import SkyRectangularAperture
except ImportError:
    flag_photu = False
else:
    flag_photu = True
```

and then only allow the `slit` option if `flag_photu=True`.

Reply:
Added changes you requested. Please see the new commit and let me know if there are additional changes that need to be made.